diff --git a/voctocore/lib/videomix.py b/voctocore/lib/videomix.py
index 38432e8..c7d29c0 100644
--- a/voctocore/lib/videomix.py
+++ b/voctocore/lib/videomix.py
@@ -95,9 +95,9 @@ class VideoMix(object):
             self.padState.append(PadState())
 
         self.log.debug('Initializing Mixer-State')
         self.compositeMode = CompositeModes.fullscreen
-        self.sourceA = 0
-        self.sourceB = 1
+        self.sourceA = 1
+        self.sourceB = 0
         self.recalculateMixerState()
         self.applyMixerState()
 
